lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -70,12 +70,19 @@
       <b>${_("No")}</b>
       %endif
     </li>
-
     <li class="field text is-not-editable" id="field-grade-cutoffs">
       <label for="start-date">${_("Grade Cutoffs:")}</label>
       <b>${ section_data['grade_cutoffs'] }</b>
     </li>
   </ul>
+
+  %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD', ''):
+      <p>
+        ## Translators: git is a version-control system; see http://git-scm.com/about
+        ${_("View detailed Git import logs for this course:")}
+        <a href="${ section_data['detailed_gitlogs_url'] }">${_("Git import logs")}</a>
```

Comment:
Sounds like a good idea to me. I amended this change to the commit.
Here's what it looks like now:
![Updated display](http://i.imgur.com/biVfLnN.png)